isaric/parsers/isaric-rapid.toml

```diff
@@ -240,6 +240,23 @@
     { field = "dsstdtc", description = "Outcome date" },
   ]
 
+[adtl.defs.followupStartDateHierarchy]
+  combinedType = "firstNonNull"
+  fields = [
```

Comment:
I think the survey dates should be linked directly to the flw_ and the flw2_ variables, rather than used in a hierarchy. So flw_survey date is used with all flw_ variables and flw2_survey_date used with all flw2_ variables. Otherwise we could end up with different data collected from 2 different surveys but with the same date, if the date is missing for one survey. This would also then be consistent with CORE follow up mapping.